nvflare/lighter/service_constants.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class FlareServiceConstants(object):
+    FLARE_CONSOLE = "admin"
+    FLARE_SERVER = "server"
+    FLARE_CLIENT = "client"
+    FLARE_OVERSEER = "overseer"
+    STARTUP = "startup"
+    CMD_START = "start"
+    CMD_STOP = "stop"
+    CMD_CLEAN = "clean"
+    EXAMPLES = "examples"
+    TRANSFER = "transfer"
+    DEFAULT_CONSOLE_USER = "admin"
```

Comment:
some of the constants are not used, maybe we can remove them.
For example: DEFAULT_CONSOLE_USER